content/applications/finance/payment_providers/stripe.rst

```diff
@@ -2,132 +2,135 @@
 Stripe
 ======
 
-`Stripe <https://stripe.com/>`_ is a United States-based online payment solution provider, allowing
+`Stripe <https://stripe.com/>`_ is a United States-based online payment solution provider allowing
 businesses to accept **credit cards** and other payment methods.
 
-Link your Stripe Account with Odoo
-==================================
+Create your Stripe account with Odoo
+====================================
 
 .. seealso::
-   - :ref:`payment_providers/add_new`
+   :ref:`payment_providers/add_new`
```

Comment:
I would also link to the PoS payment provider page for Stripe: https://www.odoo.com/documentation/master/applications/finance/payment_providers.html
I understand that this page is for online payments, but someone looking for information on a Stripe PoS terminal could easily end up on the wrong page if they just search "Stripe" and click on the wrong "Stripe" page.

Also, is there a reason this seealso is within the "Create your Stripe account with Odoo" header? I think it would be better before the header unless you have a reason you prefer it here :)